Kernel/Graphics/Intel/DisplayPortAuxiliaryConnector.cpp

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) 2022, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <Kernel/Arch/x86/IO.h>
+#include <Kernel/Graphics/Intel/DisplayPortAuxiliaryConnector.h>
+#include <Kernel/PhysicalAddress.h>
+
+namespace Kernel {
+
+enum class GMBusStatus {
+    TransactionCompletion,
+    HardwareReady
+};
+
+enum DisplayPortAuxiliary {
```

Comment:
This enum name feels like it's missing a noun. DisplayPortAuxiliary... what? DisplayPortAuxiliaryCommand? DisplayPortAuxiliaryBusState?